content/posts/building-a-successful-idp-the-human-factor.md

```diff
@@ -0,0 +1,43 @@
+
+Moreover, set up a DevOps journey that takes a developer from creating a Git project to deploying and running it in production with robust observability. Incorporate direct solutions to common questions such as:
+
+* How do I create a Git repository?
+* How do I scan my code for quality?
+* How do I deploy my application to Kubernetes?
+* Where do I store my secrets, and how do I use them in my application?
+* How can I create custom alerts?
+
+Each of these questions should lead to a clear, hands-on solution that empowers developers to act immediately and effectively.
+
+Accentuate your documentation with ready-to-go examples and a comprehensive sample application that highlights the platform’s capabilities. In doing so, you’re also creating opportunities for the platform team to dog-food their platform, iterating on their understanding from the user's perspective.
+
+## Ease of Use: The Key to Developer Happiness
+
+If your seasoned DevOps engineers are constantly referring to documentation for basic tasks, consider this a red flag for your platform's complexity. Choose tools that blend naturally with your workflow instead of forcing them to conform. Integrated tooling and process automation can greatly reduce friction; this may involve mandating the use of certain services, such as a company's docker registry, or automating deployment registrations based on organizational standards.
+
+Having a unified view of applications is crucial. Tools like Backstage consolidate information from various services into a user-friendly portal while allowing deeper dives into specific tools, all without the hassle of repeated login prompts.
+
+## Support: Expertise and Empathy
+
+Here’s the toughest nut to crack: providing responsive and effective support. Waiting for someone to address a help ticket isn't just frustrating—it can erode trust in both the IDP and the organization. Real-time help via chat or video calls signals to developers that their issues are taken seriously, fostering a supportive environment.
+
+Creating a community that nurtures peer-to-peer support can relieve some of the pressure from the platform team. This is where stellar documentation comes into play again, as it empowers users to assist each other with references and shared experiences.
+
+As your organization expands beyond 100 teams, diversifying documentation and support structures becomes necessary. Implementing tiered support, with specialist teams positioned between the developers and the platform team, can streamline assistance while maintaining a high standard.
```

Comment:
expands beyond 100 teams -> lets remove the number and do something like: expands to multitude of teams ... The number is just to indicate the larger organization but it is a strange out of nothing number now